.circleci/config.yml

```diff
@@ -78,7 +78,7 @@ jobs:
       image: << pipeline.parameters.machine_image >>
     steps:
       - checkout
-      - run: git clone https://github.com/auth0-samples/spa-quickstarts-tests test
+      - run: git clone --branch chore/update-deps https://github.com/auth0-samples/spa-quickstarts-tests test
```

Comment:
Does this branch still exist? CI failing because it doesn't exist, same as locally. Did the change get merged to main?

Reply:
Yes it did, sorry about that.